src/main/webapp/user_script.js

```diff
@@ -2,6 +2,13 @@
 * Fetches the json from ProfileServlet and displays the values on the Profile Page.
  */
 function loadProfile() {
+  $(document).ready(function() {
+    $('#navbar').load('navbar.html', function() {
+      $('#navbarDashboardSection').addClass('d-none');
+      $('#navbarProfileSection').addClass('d-none');
```

Comment:
Yea, kind of the same thing I was mentioning in the other PR, seems weird to load something and remove things. Instead would be better for everyone to load a basic navbar and your dashboard to add more stuff on top of that

Reply:
Yeah the problem with this approach is that I would still have to place some empty divs as placeholders so that the JS code knows exactly where to load the new navbar features. I decided to make the dashboard features hidden by default to try to lessen the need to hide/show different divs.